org.eclipse.jdt.debug.tests/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.debug.tests; singleton:=true
-Bundle-Version: 3.12.750.qualifier
+Bundle-Version: 3.12.850.qualifier
```

Comment:
Must be 3.12.800

Reply:
Same as above. Ideally, we should take these out of the merge work so that we get this done before people start committing other changes.